.claude/skills/README.md

```
@@ -0,0 +1,123 @@
+   ---
+   name: skillname
+   description: Brief description of what this skill does
+   trigger: skill triggers, slash command synonyms
+   ---
+   ```
+4. Document commands and workflows in the markdown body
+5. Update this README
+6. Test the skill workflow
+
+**Directory structure**:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)